### ISSUE TITLE
Feature Request: Write to Eventlog with Eventlog format

### ISSUE THREAD

**Issue by ion-storm** [fireeye/SilkETW#6]
Json is nice, however there is nested json that is hard to parse and requires select json and advanced parsing.  If possible can you log to the eventlog format so eventlog loggers like Winlogbeat parse the fields automatically.

**Comment by 0xThiebaut** [comment]
I second this. We are looking at introducing SilkETW in our course's stack but being unable to search the data retrieved through Winlogbeat is a breaker.
